{"Modified":"2019-02-27T15:21:00","Published":"2019-02-23T12:29:00","access":{"authentication":"NONE","complexity":"LOW","vector":"NETWORK"},"assigner":"cve@mitre.org","capec":[],"cvss":5,"cvss-time":"2019-02-27T15:21:00","cvss-vector":"AV:N/AC:L/Au:N/C:N/I:N/A:P","cwe":"CWE-476","id":"CVE-2019-9031","impact":{"availability":"PARTIAL","confidentiality":"NONE","integrity":"NONE"},"last-modified":"2019-02-27T15:21:00","references":["https://github.com/tbeu/matio/issues/103","https://github.com/TeamSeri0us/pocs/tree/master/matio"],"refmap":{"misc":["https://github.com/TeamSeri0us/pocs/tree/master/matio","https://github.com/tbeu/matio/issues/103"]},"summary":"An issue was discovered in libmatio.a in matio (aka MAT File I/O Library) 1.5.13. There is a NULL pointer dereference in the function Mat_VarFree() in mat.c.","vulnerable_configuration":[{"id":"cpe:2.3:a:matio_project:matio:1.5.13:*:*:*:*:*:*:*","title":"cpe:2.3:a:matio_project:matio:1.5.13:*:*:*:*:*:*:*"}],"vulnerable_configuration_cpe_2_2":[],"vulnerable_product":["cpe:2.3:a:matio_project:matio:1.5.13:*:*:*:*:*:*:*"]}